src/app/models/freshcart-categories.model.ts

```diff
@@ -0,0 +1,5 @@
+export interface FreshcartCategoriesModel {
```

Comment:
do not add "freshcart" to every model name 

The model should be named in the singular

Check in whole PR